Kitodo/src/main/java/org/kitodo/production/services/command/CommandService.java

```diff
@@ -20,13 +20,18 @@
 import java.util.List;
 import java.util.Objects;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.kitodo.api.command.CommandInterface;
 import org.kitodo.api.command.CommandResult;
+import org.kitodo.production.helper.Helper;
+import org.kitodo.production.services.data.TaskService;
 import org.kitodo.serviceloader.KitodoServiceLoader;
 
 public class CommandService {
     private final CommandInterface commandModule;
     private final ArrayList<CommandResult> finishedCommandResults = new ArrayList<>();
+    private static final Logger logger = LogManager.getLogger(TaskService.class);
```

Comment:
`logger` is not used and is using wrong log class `TaskService.class` instead of `CommandService.class` where the logger is declared.

Reply:
Yes, it is not used, thanks...will fix